.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: Continuous Integration
+
+on:
+  workflow_call:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['*']
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch (full)
+        uses: actions/checkout@v2
+
+      - name: Setup Java and Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8
+
+      - name: Cache sbt
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.coursier/cache/v1
+            ~/.cache/coursier/v1
+            ~/AppData/Local/Coursier/Cache/v1
+            ~/Library/Caches/Coursier/v1
+          key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+
+      - name: Build, test, and package project
+        run: bin/sbt clean compile test package makePom
```

Comment:
Does the cache action run again after the build?

Reply:
It seems like the cache config sets the build _to be cached_. That is, it doesn't build the cache at at the time of invocation. This was confusing to me but it's in [the example](https://github.com/actions/cache#example-workflow) before any "work" is done…